.gitignore

```diff
@@ -30,6 +30,8 @@ node_modules
 # OSX
 .DS_Store
 
+assets/emotes/*
```

Comment:
Because the emotes folder itself is empty, its not included in this PR. Because of this, when the app is started it crashes because it can't find the emotes folder. 

I would either put something in the emotes folder so it is included in repo, or create the folder if it doesn't exist prior to reading from it.

Reply:
Done. Created the folder if it doesn't exist.